aiobotocore/endpoint.py

```diff
@@ -51,43 +69,43 @@ def __dir__(self):
         return attrs
 
     def __del__(self):
-        self.close()
+        # callers may not read the Body resulting in an unclosed ClientResponse
+        if self._loop.is_running():
+            asyncio.ensure_future(self.__response.release(), loop=self._loop)
```

Comment:
here could be corner case for instance, user decided not to fetch huge file (1GB), and ensure_future with release will try to read all the data on backgound.